src/main/java/de/idrinth/waraddonclient/model/addon/ActualAddon.java

```diff
@@ -152,7 +152,7 @@ public String getInstalled() {
      * @param data
      * @return String
      */
-    private final String getStringFromObject(String key, javax.json.JsonObject data) {
+    private String getStringFromObject(String key, javax.json.JsonObject data) {
```

Comment:
while you're at it, please import `javax.json.JsonObject`

Reply:
`javax.json.JsonObject` was already imported. I pushed commit `c9db6228a02ce20ff7a8d8ca4a20393b98c6775e` to replace fully qualified class names with just the class name for any already imported classes.